.hlint.yaml

```diff
@@ -343,6 +534,70 @@
 - warn: { name: "Use 'Vector' from Universum"
         , lhs: Data.Vector.Vector, rhs: Universum.Vector }
 
+## Deepseq
+- warn: { name: "Use 'NFData' from Universum"
+        , lhs: Control.DeepSeq.NFData, rhs: Universum.NFData }
+- warn: { name: "Use 'rnf' from Universum"
+        , lhs: Control.DeepSeq.rnf, rhs: Universum.rnf }
+- warn: { name: "Use 'deepseq' from Universum"
+        , lhs: Control.DeepSeq.deepseq, rhs: Universum.deepseq }
+- warn: { name: "Use 'force' from Universum"
+        , lhs: Control.DeepSeq.force, rhs: Universum.force }
+- warn: { name: "Use '($!!)' from Universum"
+        , lhs: "Control.DeepSeq.($!!)", rhs: "Universum.($!!)" }
+
+## Exception
+- warn: { name: "Use 'Exception' from Universum"
+        , lhs: Control.Exception.Safe.Exception, rhs: Universum.Exception }
+- warn: { name: "Use 'toException' from Universum"
+        , lhs: Control.Exception.Safe.toException, rhs: Universum.toException }
+- warn: { name: "Use 'fromException' from Universum"
```

Comment:
I think it's better to add `Exception`, `fromException` and `toException` rules for `Control.Exception` module as well because not everybody uses `safe-exceptions` library:

* http://hackage.haskell.org/package/base-4.10.1.0/docs/Control-Exception.html